wp-content/themes/orient-theme/taxonomy-series.php

```diff
@@ -1,18 +1,35 @@
 <?php
 
-get_header(); ?>
+get_header();
 
+//create a copy of wp_query
+$copy = clone $wp_query;
 
+//alternative???
+// $copy = WP_Query($wp_query);  
+// $copy = new WP_Query( array('post_type' => 'post') );
+$copy->set('nopaging', true);
+$wp_query->set( 'nopaging' , true );
 
+
+//find the date for the first and latest articles
+$first_article_date = get_the_date('F j, Y', $copy->posts[$copy->post_count - 1]);
+$latest_article_date = get_the_date('F j, Y', $copy->posts[0]);
+
+// reset query ... doesn't work??
```

Comment:
If there's a function that doesn't work, I don't think it should necessarily be included in the PR.

Reply:
Got it! I read through all the comments you left & I'll be sure to implement all those in future projs. I see what you mean when you say this is where the meaningful discussion about code happens!!